From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sun, 23 Jun 2024 03:29:43 +0200
Subject: [PATCH] tests: Drop check-format test

It modifies the Git tree and prevents patches from being applied
correctly.

This really should be part of a pre-commit check, not a Meson test. The
source directory should not be modified.
---
 tests/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index ff9004d6ad75..178a2f09f1aa 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -1,4 +1,3 @@
-test('check-format', find_program('check-format.sh'))
 
 libmocklibc = subproject('mocklibc').get_variable('libmocklibc')
 
